### PR TITLE
[full-ci] Bump Reva

### PR DESCRIPTION
**Pull request by kobergj**
Bumps reva

Fixes https://github.com/owncloud/ocis/issues/5139
Fixes https://github.com/owncloud/ocis/issues/6555

### CODE REVIEW

**Comment by phil-davis** [comment]
https://drone.owncloud.com/owncloud/ocis/23546/49/5
```
  Scenario: download a single file as different user                                                  # /drone/src/tests/acceptance/features/apiArchiver/downloadById.feature:69
    Given user "Brian" has been created with default attributes and without skeleton files            # FeatureContext::userHasBeenCreatedWithDefaultAttributesAndWithoutSkeletonFiles()
    And user "Alice" has uploaded file with content "some data" to "/textfile0.txt"                   # FeatureContext::userHasUploadedAFileWithContentTo()
    When user "Brian" downloads the archive of "/textfile0.txt" of user "Alice" using the resource id # ArchiverContext::userDownloadsTheArchiveOfItemOfUser()
    Then the HTTP status code should be "404"                                                         # FeatureContext::thenTheHTTPStatusCodeShouldBe()
      HTTP status code 200 is not the expected value 404
      Failed asserting that 200 matches expected '404'.
```

It seems that Brian is now able to access a file that belongs to Alice. That needs to be investigated - is it the server code, or is the test doing something unexpected?

**Comment by kobergj** [comment]
Kind of. He gets the `not found` error message with no `404` status code but `200` instead. I'm investigating...

**Comment by kobergj** [comment]
Needs another reva fix unforty: https://github.com/cs3org/reva/pull/4010

**Comment by phil-davis** [comment]
https://github.com/kobergj/ocis/tree/BumpReva
This branch is [2 commits ahead](https://github.com/kobergj/ocis/compare/owncloud:ocis:master...BumpReva), [13 commits behind](https://github.com/kobergj/ocis/compare/BumpReva...owncloud:ocis:master) owncloud:master.

I'm not sure if there was anything important in the most recent 13 commits, but I think that GitHub does an auto-merging anyway when it provides the branch contents to drone CI.

**Comment by kobergj** [comment]
You are right as always :smile:  I rebased to current master to minimize risk.

File: changelog/unreleased/bump-reva.md [hunk 1]
```diff
@@ -28,3 +28,4 @@ https://github.com/owncloud/ocis/pull/6529
 https://github.com/owncloud/ocis/pull/6544
 https://github.com/owncloud/ocis/pull/6507
 https://github.com/owncloud/ocis/pull/6572
+https://github.com/owncloud/ocis/pull/6589
```
Comment (phil-davis) [inline changelog/unreleased/bump-reva.md:31]:
```suggestion
https://github.com/owncloud/ocis/pull/6590
```

**Review by phil-davis — APPROVED** [review]
LGTM if CI passes.

**Pull request merged**